packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -66,6 +68,8 @@ MeetingInfoUtil.isMeetingLink = (value: string) => {
     parsedUrl.pathname &&
     (parsedUrl.pathname.includes(`/${MEET}`) ||
       parsedUrl.pathname.includes(`/${MEET_M}`) ||
+      parsedUrl.pathname.includes(`/${MEET_CISCO}`) ||
```

Comment:
Shouldn't we add tests for these changes?

Reply:
As I mentioned in the space, this change is temporary. We might change this again. But if we need test written for the temporary change, I can still go ahead. 

I am following up constantly with Ask App API team in the space for the right way to do it. This change is just until they come back with answers